spec/features/state_file/complete_intake_spec.rb

```diff
@@ -492,12 +492,21 @@
       # 1099G Review
       click_on I18n.t("general.continue")
 
+      # Health Insurance Premium
+      expect(page).to have_text I18n.t('state_file.questions.id_health_insurance_premium.edit.title')
+      choose I18n.t("general.affirmative")
+      fill_in 'state_file_id_health_insurance_premium_form_health_insurance_paid_amount', with: "1234.60"
+      click_on I18n.t("general.continue")
+
+      # Sales/Use Tax
       expect(page).to have_text I18n.t('state_file.questions.id_sales_use_tax.edit.title', year: MultiTenantService.statefile.current_tax_year)
       choose I18n.t("general.affirmative")
       fill_in 'state_file_id_sales_use_tax_form_total_purchase_amount', with: "290"
+      click_on I18n.t("general.continue")
 
       # ID Review page
-      click_on I18n.t("general.continue")
+      expect(page).to have_text "Idaho Health Insurance Premium Subtraction"
+      expect(page).to have_text "$1,234.60"
 
```

Comment:
hmm wondering if going forward we should all add the review page content

Reply:
yes i think we should